include/itkMetamorphosisImageRegistrationMethodv4.h

```diff
@@ -67,8 +67,8 @@ public ImageToImageFilter<TInputImage, TOutputImage>
 
 
 protected:
-  ConstantImageFilter(){}
```

Comment:
We should remove `override` from the constructor.

Reply:
Thanks for keeping an eye. Addressed in  9160b83.